week-2-practise/level-2/1-boolean.js

```diff
@@ -3,11 +3,12 @@
 // Fix it.
 
 var isHappy = "false";
+var isHappy = "true";
```

Comment:
There are now two declarations for `isHappy`. You should just have one. What type is `"true"` and what should it be here?